docs/pages/docs/walkthroughs/lesson-1.mdx

```diff
@@ -0,0 +1,100 @@
+import { Markdown } from '../../../components/Markdown';
+
+# Lesson 1: Setting up Keystone
+
+> In this lesson we will install the keystone package, create our first list, and get Keystone running
+
+## Installing the packages you need
+
+We will be using `yarn` for this, however you can use `npm` if you would prefer.
+
+The first thing we are going to do is set ourselves up a workspace for our new project. Let's make ourselves a new folder and initialise yarn.
+
+```sh
+mkdir keystone-learning
+cd keystone-learning
+yarn init
+```
+
+After this, let's add the keystone package
+
+```sh
+yarn add @keystone-6/keystone
+```
+
+## Setting up Keystone
+
+There are two bits of keystone's configuration that are required to start our project: the `lists`, and the `db`. The `lists` define
+she shape of the information we want to save in our database, while the `db` defines how we will connect to our database. For now though
+let us set up our `keystone.js` file
+
+`touch keystone.js`
```

Comment:
Probably don't need to `touch`,   I can't think of many editors that won't create the file if it doesn't exist when saving